basket/news/tasks.py

```diff
@@ -256,10 +256,25 @@ def fxa_email_changed(data):
         # message older than our last update for this UID
         return
 
+    # Update SFDC
+    user_data = get_user_data(fxa_id=fxa_id, extra_fields=['id'])
+    if user_data:
+        sfdc.update(user_data, {'fxa_primary_email': email})
+    else:
+        # FxA record not found, try email
+        user_data = get_user_data(email=email, extra_fields=['id'])
+        if user_data:
+            sfdc.update(user_data, {'fxa_id': fxa_id})
```

Comment:
Hey @mozbrandon @andrewmozilla can you please advise on whether we should also set `FxA_Primary_Email__c` in this case, i.e. when we find the user in SFDC only by `email` and they have no `FxA ID` assigned.

Reply:
Yes, please set `FxA_Primary_Email__c` in this case.